ui/src/wizard/AzureWizard.ts

```diff
@@ -72,16 +82,26 @@ export class AzureWizard<T extends IInternalActionContext> implements types.Azur
                 if (step.shouldPrompt(this._context)) {
                     step.propertiesBeforePrompt = Object.keys(this._context).filter(k => !isNullOrUndefined(this._context[k]));
 
+                    const loadingQuickPick = this._showLoadingPrompt ? createQuickPick(this._context, {
```

Comment:
Let's check `cancellationToken.isCancellationRequested` in a few more spots, at least once per `await` call. I would suggest
1. At the beginning of the loop right after `while (step) `
1. Right before `await step.getSubWizard`